instrumentation/graphql-java-12.0/library/src/main/java/io/opentelemetry/instrumentation/graphql/GraphQLTelemetryBuilder.java

```diff
@@ -13,24 +13,12 @@ public final class GraphQLTelemetryBuilder {
 
   private final OpenTelemetry openTelemetry;
 
-  private boolean captureExperimentalSpanAttributes;
   private boolean sanitizeQuery = true;
 
   GraphQLTelemetryBuilder(OpenTelemetry openTelemetry) {
     this.openTelemetry = openTelemetry;
   }
 
-  /**
-   * Sets whether experimental attributes should be set to spans. These attributes may be changed or
-   * removed in the future, so only enable this if you know you do not require attributes filled by
-   * this instrumentation to be stable across versions.
-   */
-  public GraphQLTelemetryBuilder setCaptureExperimentalSpanAttributes(
```

Comment:
typically we have marked as `@Deprecated` for one release cycle before removing

Reply:
good point, added the method back and marked as `@Deprecated`